sunpy/net/dataretriever/sources/fermi_gbm.py

```diff
@@ -111,6 +113,13 @@ def _can_handle_query(cls, *query):
                 return all(chklist)
         return False
 
+    @classmethod
+    def register_values(cls):
+        from sunpy.net import attrs
+        adict = {attrs.Instrument: [
```

Comment:
Why we are not registering `a.Resolution` and `a.Detector` for `fermi_gbm`?

Reply:
An oversight on my part, it should be there.

Reply:
Fine, we don't check what resolution and Detector can possibly be. We only check that they should be there in the `can_handle_query`.

Reply:
True, I am not sure how I would register that however.

Reply:
```python
	adict = {attrs.Instrument: [('GBM', 'Gamma-Ray Burst Monitor on board the Fermi satellite.')],
		     attrs.Physobs: [('CSPEC' , 'counts accumulated every  4.096 seconds in 128 energy channels for each detector.'),
					      ('CTIME', 'counts accumulated every 0.256 seconds in 8 energy channels')],
		     attrs.Resolution: [('N'+str(x), f"GBM Detector short name for the detector NAI_{y:02}") for x in range(12)]
		     }
        return adict
```
@nabobalis This will do it according to what our client checks.

Reply:
Added.